airflow/providers/amazon/aws/operators/emr_containers_get_job_state.py

```diff
@@ -0,0 +1,61 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from typing import Any, Dict
+
+from airflow.models import BaseOperator
+from airflow.providers.amazon.aws.hooks.emr_containers import EmrContainersHook
+
+
+class EmrContainersGetJobStateOperator(BaseOperator):
+    """Operator to get a job status.
+
+    A job run is a unit of work, such as a Spark jar, PySpark script, or SparkSQL query,
+    that you submit to Amazon EMR on EKS.
+
+    A job is in PENDING, SUBMITTED, RUNNING, FAILED, CANCELLED or CANCEL state
```

Comment:
```suggestion
    A job is in PENDING, SUBMITTED, RUNNING, FAILED, CANCELLED, CANCEL_PENDING, or COMPLETED state
```

Per https://docs.aws.amazon.com/emr-on-eks/latest/APIReference/API_JobRun.html